packages/signal-polyfill/src/wrapper.ts

```diff
@@ -206,7 +206,7 @@ export namespace subtle {
     constructor(notify: (this: Watcher) => void) {
       let node = Object.create(REACTIVE_NODE);
       node.wrapper = this;
-      node.consumerMarkedDirty = notify;
+      node.consumerMarkedDirty = () => notify.call(this);
```

Comment:
Could you fix up [graph.ts line 338](https://github.com/proposal-signals/proposal-signals/blob/08a210bb501eebe4b71c0ae0c856b731e93a2c23/packages/signal-polyfill/src/graph.ts#L338) instead?

Reply:
it should pass in node.wrapper as this, rather than node as the argument